jest.config.js

```diff
@@ -20,8 +20,8 @@ module.exports = {
   coverageThreshold: {
     global: {
       statements: 95,
-      branches: 94.23,
```

Comment:
sorry I don't really have that information, but is it ok to reduce the threshold?

Reply:
To my understanding these were just arbitrary limits set by the previous team. 94.23 is a weird number to have set it to and lowering it whilst resources are thin seemed like the best solution.

Reply:
I see